src/_codelabs/index.md

```diff
@@ -8,7 +8,14 @@ toc: false
 
 See the latest Dart codelabs on the Google Developers site:
 
+**General**
 * [Intro to Dart for Java Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart)
+
+**Flutter**
+* [Building Beautiful UIs with Flutter](https://codelabs.developers.google.com/codelabs/flutter)
+* [Firebase for Flutter](https://codelabs.developers.google.com/codelabs/flutter-firebase)
+
+**AngularDart**
 * [Write a Material Design AngularDart Web App](https://codelabs.developers.google.com/codelabs/your-first-angulardart-web-app)
```

Comment:
Looks good!

We no longer need the "Other codelabs" section, right? Just a final sentence linking to Dart for the Web codelabs, and we're good.